src/test/regress/sql/upgrade_post_11_after.sql

```diff
@@ -0,0 +1,11 @@
+SET search_path = post_11_upgrade;
+
+-- make sure that we always (re)sync the metadata
+UPDATE pg_dist_node_metadata SET metadata=jsonb_set(metadata, '{partitioned_citus_table_exists_pre_11}', to_jsonb('true'::bool), true);
+SELECT citus_finalize_upgrade_to_citus11(enforce_version_check:=false);
+
+-- tables are objects with Citus 11+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) FROM citus.pg_dist_object WHERE objid IN ('post_11_upgrade'::regnamespace, 'post_11_upgrade.part_table'::regclass, 'post_11_upgrade.sensors'::regclass, 'post_11_upgrade.func_in_transaction_def'::regproc, 'post_11_upgrade.partial_index_test_config'::regconfig, 'post_11_upgrade.my_type'::regtype) ORDER BY 1;
+
```

Comment:
it'd be nice to also use a metadata sanity check here. (do all nodes produce the same metadata snapshot)

Reply:
makes sense

Reply:
It seems like the check produces the same result, but the object creation command is not sorted in the same order as the OIDs differ across nodes. The command I mention is the following. 

```SQL
WITH distributed_object_data(typetext, objnames, objargs, distargumentindex, colocationid, force_delegation)  AS (VALUES ('type', ARRAY['fooschema.footype']::text[], ARRAY[]::text[], -1, 0, false), ....., ('text search configuration', ARRAY['post_11_upgrade', 'partial_index_test_config']::text[], ARRAY[]::text[], -1, 0, false)) SELECT citus_internal_add_object_metadata(typetext, objnames, objargs, distargumentindex::int, colocationid::int, force_delegation::bool) FROM distributed_object_data;
```

So, I exclude this specific command, which should be fine because all other objects are already exists on the snapshot (e.g., create trigger etc)